setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
I tried playing with RTD on the weekend, but didn't have any luck. It seems Sphinx needs to have the extensions built to get the docstrings, but RTD doesn't let one build the extensions. One option would be to mock out the Cython parts when building the docs. Not ideal though. 😞